From 8afadea98e87e14297969dbc2a75cc9e938a502b Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 22 Nov 2024 18:01:24 +0100
Subject: [PATCH 713/954] fix(stm32mp21): remove C1P1 flag in RCC

Remove C1P1RSTF flag in RCC_C1HWRSTSCLRR, RCC_C1BOOTRSTSSETR
and RCC_C1BOOTRSTSCLRR

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git b35ab6f4600a5f0d25a5ef39ecd451cd08179eef fix(stm32mp21): remove C1P1 flag in RCC]
Fixes: f74e4e80690a ("feat(stm32mp21): add RCC registers file")
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I47c9fa15db83bec599423fa0f8f2f7457c530902
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/420549
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/438134
Domain-Review: Maxime MERE <maxime.mere@foss.st.com>
Tested-by: Maxime MERE <maxime.mere@foss.st.com>
---
 include/drivers/st/stm32mp21_rcc.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/include/drivers/st/stm32mp21_rcc.h b/include/drivers/st/stm32mp21_rcc.h
index 6eac9fc7bf..2067bec7f8 100644
--- a/include/drivers/st/stm32mp21_rcc.h
+++ b/include/drivers/st/stm32mp21_rcc.h
@@ -2208,7 +2208,6 @@
 /* RCC_C1HWRSTSCLRR register fields */
 #define RCC_C1HWRSTSCLRR_VCPURSTF		BIT(0)
 #define RCC_C1HWRSTSCLRR_C1RSTF			BIT(1)
-#define RCC_C1HWRSTSCLRR_C1P1RSTF		BIT(2)
 
 /* RCC_C2HWRSTSCLRR register fields */
 #define RCC_C2HWRSTSCLRR_C2RSTF			BIT(0)
@@ -2227,7 +2226,6 @@
 #define RCC_C1BOOTRSTSSETR_IWDG3SYSRSTF		BIT(10)
 #define RCC_C1BOOTRSTSSETR_IWDG4SYSRSTF		BIT(11)
 #define RCC_C1BOOTRSTSSETR_C1RSTF		BIT(13)
-#define RCC_C1BOOTRSTSSETR_C1P1RSTF		BIT(16)
 #define RCC_C1BOOTRSTSSETR_RETCRCERRRSTF	BIT(17)
 #define RCC_C1BOOTRSTSSETR_RETECCFAILCRCRSTF	BIT(18)
 #define RCC_C1BOOTRSTSSETR_RETECCFAILRESTRSTF	BIT(19)
@@ -2249,7 +2247,6 @@
 #define RCC_C1BOOTRSTSCLRR_IWDG3SYSRSTF		BIT(10)
 #define RCC_C1BOOTRSTSCLRR_IWDG4SYSRSTF		BIT(11)
 #define RCC_C1BOOTRSTSCLRR_C1RSTF		BIT(13)
-#define RCC_C1BOOTRSTSCLRR_C1P1RSTF		BIT(16)
 #define RCC_C1BOOTRSTSCLRR_RETCRCERRRSTF	BIT(17)
 #define RCC_C1BOOTRSTSCLRR_RETECCFAILCRCRSTF	BIT(18)
 #define RCC_C1BOOTRSTSCLRR_RETECCFAILRESTRSTF	BIT(19)
-- 
2.53.0

